src/mscorlib/src/System/Numerics/Hashing/HashHelpers.cs

```diff
@@ -8,12 +8,14 @@ namespace System.Numerics.Hashing
 
     internal static class HashHelpers
     {
+        public static readonly int RandomSeed = Guid.NewGuid().GetHashCode();
```

Comment:
This does not seem to be actually used.

Also, could you please change this to be initialized with `new Random().Next(Int32.MinValue, Int32.MaxValue)` instead? CoreCLR and CoreRT are initializing the random seed via NewGuid() already - it would be nice to do it just once per process.